oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -189,6 +189,32 @@ reasons that we impact site operators (new translations location), we will have
 development tools as well. In addion, we will create new instructions for developers on
 how to enable translations for a new service/repo when it comes online.
 
+Final State
+***********
+
+Repositories that generate translation files will have their translation files generated
+and pushed to the openedx-translation repository via a github workflow. Once the
+translation files from edx-platform and other repositories are moved to the
+openedx-translations repository, the Transifex GitHub App will link a Transifex project
+of a name such as "Open edX Translations" to the openedx-translations repository. A
+configuration file naming the files that are to be translated and the trigger that pulls
+translation files back into will be created in the openedx/translations repository. This
+link will allow for the Transifex GitHub App to automatically manage the push/pull of the
+translation files without the need for human intervention.
+
+When it comes time to cut an Open Release, a new branch will be formed in the
+openedx-translations repository for this release. This new branch will correspond with
+other release branches such as those found in edx-platform. It is my recommendation that
+a tool in the form of a python library is written to enable the placement of the
+translation files kept in openedx-translations into the repositories the translation
+files are formed from. This tool will manage the placement of translation files through
```

Comment:
It sounds like this tool would be used at release time? But doesn't it need to do the placement and concatenation o of translation files for every deploy?

Reply:
Oh, you said that below.